doc/mkdocs/docs/api/basic_json/operator[].md

```diff
@@ -70,7 +77,8 @@ Strong exception safety: if an exception occurs, the original value stays intact
 
 1. Constant if `idx` is in the range of the array. Otherwise, linear in `idx - size()`.
 2. Logarithmic in the size of the container.
-3. Constant
+3. See 2.
```

Comment:
Nitpicky, but I'd rather repeat `Logarithmic in the size of the container.` here. The level of indirection for that short sentence confuses me.

Reply:
While we're on the subject of complexities: Shouldn't any access via `json_pointer` be at least linear? `O(N)` with `N` being the number of reference tokens. And that ignores the complexity of each access per reference token.

Reply:
Constant makes no sense here. It's logarithmic in the size of the container (because of the way reference tokens are interpreted for objects; a JSON pointer that only traverses through arrays like `/0/1/2/3` would be constant).

"O(N) with N being the number of reference tokens" is true, but the length of a JSON Pointer should be negligible compared to the size of a container.

Reply:
The documentation apparently needs more eyes on it...

Reply:
@nlohmann Because I just noticed that I'm missing an exception description for an overload: A more advanced feature of `check_structure.py` could be to check if sections like "Return value", "Exceptions", "Complexity" use numbered lists, that the number of entries is the same.

(This is also an example of something I might put into `Roadmap.md`.)

Reply:
Good point. Though I still think issues or discussions are a better place to collect such thoughts...

Reply:
The issue templates basically allow anything. Past experiences however made me request a lot of standardized information, because people tend to be lazy and create issues where a lot of back and forth is needed to understand what it's about.

Reply:
See #3455. I think Discussion/Ideas are the format we need.